src/server/common/components/site-details-map/template.njk

```diff
@@ -0,0 +1,2 @@
+<link href="/public/stylesheets/ol.css" rel="stylesheet">
```

Comment:
if there are multiple maps, the stylesheet link will be rendered multiple times? I guess it will be cached